bulletroboy/cage_simulation.py

```diff
@@ -28,20 +31,47 @@
     for tendon in exoforce.get_tendons():
         tendon.forceId = p.addUserDebugParameter("Force in " + tendon.name, 0, 200, 0)
 
-    cage_angle_id = p.addUserDebugParameter("Cage Angle", -180, 180, 0)
+    automatic_cage_rotation = True
+    issue = False
```

Comment:
what is issue?

Reply:
I changed the name issue to "error" I think it is more intuitive. This variable is set to False and is turned to True by the movement functions if they encounter a problem. Using the if statement within the while loop, the figure will only be moved if there is no error (error = False).

Reply:
At the beginning the movement functions gave several errors and crashed the sim. I used the error variable to check if the movements would be problematic and to stop the sim from crashing. I tested it all again and I see that it is no longer necessary. So I deleted that implementation. No error or issue variable now 👍